src/schema/v2/user.ts

```diff
@@ -43,6 +43,24 @@ export const UserAdminNoteType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLString),
     },
     createdAt: date(({ created_at }) => created_at),
+    adminName: {
+      description: "The name of the admin user who created the note",
+      type: GraphQLString,
+      resolve: async ({ created_by }, {}, { userByIDLoader }) => {
+        if (!userByIDLoader) {
+          throw new Error(
+            "Loader not found. You must supply an X-Access-Token header."
```

Comment:
I am unsure as far as this message is concerned. On the `user` we have two forms of this message:

> You need to pass a X-Access-Token header to perform this action

and 

> Loader not found. You must supply an X-Access-Token header.

Which one should be used then?

Reply:
Should we use one form in this file? I am happy to update all other lines so that it's a consistent message.

Reply:
I think it is a good idea to standardize this. Would even suggest just saying `An X-Access-Token header to perform this action.`

Reply:
Made the change in this commit https://github.com/artsy/metaphysics/pull/4515/commits/0266ea806dc70c752fc62a59ee26e9b7d7e6cd3b to use the same message across the file:
> An X-Access-Token header is required to perform this action.

Reply:
😬 Put up [a33edaa](https://github.com/artsy/metaphysics/pull/4515/commits/a33edaa0a2bf53c110fdfc032cf19c9fcaf2e57b) to fix my own grammar mistake.